recipes/quantum-kite/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "quantum-kite" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ejh516/kite/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 1ea3875f61ff30785d5742aaede71fa519bb4440a2b961f368a8f34ec92b4136
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - hdf5
+    - eigen
+    - pip
+  run:
+    - pybinding
+    - {{ pin_compatible('numpy') }}
```

Comment:
You are pinning a compatible numpy, but you don't include numpy as a build dependency? You don't need to pin compatible if you aren't building against numpy.